post_installation_configuration/aws-compute-edge-tasks.adoc

```diff
@@ -0,0 +1,73 @@
+:_content-type: ASSEMBLY
+[id="aws-compute-edge-tasks"]
+= AWS Local Zone tasks
+include::_attributes/common-attributes.adoc[]
+:context: aws-compute-edge-tasks
+
+toc::[]
+
+After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS Local Zones and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
```

Comment:
We rarely say "to meet your needs." I'll let you decide whether to keep it, but the comma isn't needed:

```suggestion
After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS Local Zones and an edge compute pool so that you can expand and customize your cluster.
```